server/cache.go

```diff
@@ -336,6 +348,18 @@ func (c *clusterInfo) putStoreLocked(store *storeInfo) error {
 	return nil
 }
 
+func (c *clusterInfo) blockStore(storeID uint64) {
+	c.Lock()
+	defer c.Unlock()
```

Comment:
I think it is more consistent to use `defer` here, no matter it is simple or not.
Otherwise, every time I use a lock, I need to consider whether it is simple enough that I don't need to `defer`.